docs/openapi.yaml

```diff
@@ -0,0 +1,1143 @@
+openapi: 3.0.1
+info:
+  title: "Flotilla API"
+  version: "1.0.0"
+  contact:
+    email: "fg_robot_dev@equinor.com"
+
+  license:
+    name: Eclipse 2.0
+    url: https://www.eclipse.org/org/documents/epl-2.0/EPL-2.0.txt
+
+  description: |
+    The Flotilla API is an open API covering the robots in use on an asset within Equinor.
+    This document describes release v1.0.0. We recommend using documentation published at [github.io](https://equinor.github.io/api-first-workbench) as it provides the best developer experience.
```

Comment:
I suggest a link to the readme and not say version number in this text. Then in the readme link to the hosted version of this specification.